### PR TITLE
[PM-7662] Suppress welcome window on install when extension is in dev mode

### PR DESCRIPTION
**Pull request by jprusik**
## Type of change

- Other

## Objective

Don't show the welcome window every time the extension is reinstalled when debugging

### CODE REVIEW

**Review by MGibson1 — CHANGES_REQUESTED** [review]
Sorry for the bad communication, but this isn't what I meant. the section o the code I linked define compile time flags that are never true in prod environments, but are configurable in dev environments.

I want to be able to easily turn this back on should we need to test

File: apps/browser/config/base.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "dev_flags": {},
```
Comment (jprusik) [inline apps/browser/config/base.json:2]: This and the update in desktop base config are presumptive given no apparent consumption of this casing pattern.

Screenshot 2024-05-01 at 11 21 03 AM

Comment (MGibson1) [reply apps/browser/config/base.json:2]: yep, should be camel case, thank you

**Review by MGibson1 — APPROVED** [review]
nice, thank you!

**Pull request merged**